ci/cron/daily-compat.yml

```diff
@@ -234,16 +223,6 @@ jobs:
         fi
 
         sed -i 's|SKIP_DEV_CANTON_TESTS=.*|SKIP_DEV_CANTON_TESTS=false|' build.sh
-        sed -e 's/^/# /' COPY > canton_dep.bzl
-        cat <<EOF >> canton_dep.bzl
-
-        canton = {
```

Comment:
As we're not touching this file anymore, in the spirit of cleaning things up and not leaving false trails for future readers, you probably also want to remove `canton_dep.bzl` from lines 263 and 267.

Alternatively, maybe this should be kept in part and become:
```bash
cat <<EOF >> canton_dep.bzl
canton = {
    "local": False,
}
```
so the daily PR resets this in case someone had committed a "local" build.

Reply:
Thanks for catching those, I've removed them. If someone as committed a local build, I think the CI would fail so they shouldn't be able to merge it.